packages/react-composites/src/composites/CallComposite/components/CallControls.tsx

```diff
@@ -111,9 +113,17 @@ export const CallControls = (props: CallControlsProps & ContainerRectProps): JSX
       setDeepNoiseSuppressionOn(true);
     }
   }, [adapter, startDeepNoiseSuppression]);
+
+  /* @conditional-compile-remove(DNS) */
+  const environmentInfo = adapter.getState().environmentInfo;
```

Comment:
Approving because environment doesn't change dynamically during the call so does not impact this PR - but we should be using selectors anytime we are accessing adapter state so that we correctly re-render when changes occur